core/frontend/src/EmphasizeElements.ts

```diff
@@ -32,17 +32,19 @@ export class EmphasizeElements implements FeatureOverrideProvider {
     const emphasizedElements = this.getEmphasizedElements(vp);
     if (undefined !== emphasizedElements) {
       overrides.setDefaultOverrides(this._defaultAppearance!);
-      const app = this.wantEmphasis ? this._emphasizedAppearance : FeatureAppearance.defaults;
-      emphasizedElements.forEach((id) => { overrides.overrideElement(id, app); });
+      const appearance = this.wantEmphasis ? this._emphasizedAppearance : FeatureAppearance.defaults;
+      for (const elementId of emphasizedElements)
+        overrides.override({ elementId, appearance });
```

Comment:
this is going to create an object for every iteration. Should you re-use an object?

Reply:
I wrote a little performance test that shows reusing an object is ~25% faster so I guess so.

Reply:
Pushed.